benches/generators.rs

```diff
@@ -45,9 +45,11 @@ macro_rules! gen_uint {
         fn $fnn(b: &mut Bencher) {
             let mut rng = $gen;
             b.iter(|| {
+                let mut accum: $ty = 0;
                 for _ in 0..RAND_BENCH_N {
-                    black_box(rng.gen::<$ty>());
+                    accum = accum.wrapping_add(rng.gen::<$ty>());
                 }
+                black_box(accum);
```

Comment:
I had on my wish-list to investigate the benchmark 'problems'. So nice to see 😄. I also noticed some change if you set `b.bytes` before `b.iter`. Is that something you can confirm?

Reply:
Never tried that.

Something similar might be possible for the `fill_bytes` stuff, but it's probably less significant (already this has little impact on 64-bit outputs).

Reply:
It made a difference, but not any more with the accumulator.